cms/settings/base.py

```diff
@@ -975,7 +975,7 @@
 SEARCH_INDEX_PUBLISHER_BACKEND = os.getenv("SEARCH_INDEX_PUBLISHER_BACKEND")
 KAFKA_SERVERS = os.getenv("KAFKA_SERVERS", "").split(",")
 KAFKA_USE_IAM_AUTH = os.getenv("KAFKA_USE_IAM_AUTH", "false").lower() == "true"
-KAFKA_API_VERSION = tuple(map(int, os.getenv("KAFKA_API_VERSION", "3,5,1").split(",")))
+KAFKA_API_VERSION = tuple(map(int, os.getenv("KAFKA_API_VERSION", "3.5.1").split(".")))
```

Comment:
https://github.com/ONSdigital/dis-wagtail/blob/main/docs/integrations/search_service.md?plain=1#L21 needs updating, other than that LGTM

Reply:
Done! [011a403](https://github.com/ONSdigital/dis-wagtail/pull/300/commits/011a40370ead48756cc4c4994ceb1ee15f0c8474)